### PR TITLE
[Mobile]Fix placeholder position of block appender

### PR DESCRIPTION
**Pull request by pinarol**
## Description
To fix: https://github.com/wordpress-mobile/gutenberg-mobile/issues/637

Updated block appender to match the placeholder position of paragraph block.

## How has this been tested?

Tested with steps on https://github.com/wordpress-mobile/gutenberg-mobile/pull/732

## Checklist:
- [x] My code is tested.
- [x] My code follows the WordPress code style. 
- [x] My code follows the accessibility standards. 
- [x] My code has proper inline documentation. 
- [x] I've included developer documentation if appropriate.

### CODE REVIEW

**Review by etoledom — APPROVED** [review]
It looks and works perfectly 🎉

I agree that the most sane way of "reusing" the style here is to use the same component.

I feel a bit sad though, seeing that we use `rich-text` (and also a full Aztec instance) just to show the styled placeholder. But since it's only present when there's no content, it shouldn't be an issue.

I left a couple of small code comments, let's :shipit: when ready! 👍

File: packages/editor/src/components/default-block-appender/index.native.js [hunk 1]
```diff
@@ -33,13 +35,9 @@ export function DefaultBlockAppender( {
...
 		>
 			<View style={ styles.blockHolder } pointerEvents="box-only">
-				<View style={ styles.blockContainer }>
-					<TextInput
-						style={ styles.textView }
-						textAlignVertical="top"
-						multiline
-						numberOfLines={ 0 }
-						value={ value }
+				<View style={ containerStyle }>
```
[the remark this answers is not in the document]
Comment (pinarol) [reply packages/editor/src/components/default-block-appender/index.native.js]: Done. It didn't break anything so I think we're better this way 👍

File: packages/editor/src/components/default-block-appender/style.native.scss [hunk 2]
```diff
@@ -6,16 +6,3 @@
...
-	background-color: $white;
-	padding-top: 0;
-	padding-left: 16px;
-	padding-right: 16px;
-}
-
-.textView {
-	color: $gray;
-	font-size: 16px;
-	font-family: $default-regular-font;
```
[the remark this answers is not in the document]
Comment (pinarol) [reply packages/editor/src/components/default-block-appender/style.native.scss]: Done

**Comment by pinarol** [comment]
@etoledom thanks for the review, I just did your suggestions. You can have another look if you want

**Comment by etoledom** [comment]
Thank you!
:shipit: 

Not sure about what to do with Travis though 🤔

**Comment by pinarol** [comment]
> I feel a bit sad though, seeing that we use rich-text (and also a full Aztec instance) just to show the styled placeholder. But since it's only present when there's no content, it shouldn't be an issue.

@etoledom this way we won't have to touch block appender when any one of the platforms make a change in aztec in the future. if we'd continued using TextInput we'd need to adjust its position for both platforms separately(because both platforms have separate insets in aztec currently). at the end of the day we want block appender to look exactly like a paragraph block so we are saving ourselves doing the maintenance of keeping it that way.

**Pull request merged**